modules/misc/doc/content/modules/misc/index.md

```diff
@@ -5,4 +5,4 @@ Documentation for the misc module needs some work...
 
 ## Objects, Actions, and Syntax
 
-!syntax complete group=MiscApp level=3
+!syntax complete groups=MiscApp level=3
```

Comment:
Why do some of these apps have level=3 while others use the default level=2?

Reply:
Whoever created the docs must of wanted it that way. Not sure.